tests/db/repositories/test_run_repository.py

```
@@ -0,0 +1,819 @@
+            run_id=run_id,
+            created_at="2024_01_01-12:00:00",
+            total_turns=10,
+            total_agents=5,
+            started_at="2024_01_01-12:00:00",
+            status=RunStatus.COMPLETED,
+            completed_at="2024_01_01-13:00:00"
+        )
+        
+        with patch.object(repo, "get_run", return_value=current_run):
+            # Act & Assert
+            with pytest.raises(ValueError) as exc_info:
```

Ruff (0.14.6)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)